### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by vweevers**
Started with some cleanup.

### CODE REVIEW

**Comment by vweevers** [comment]
Node 10 is failing as expected, due to outdated leveldown.

**Comment by vweevers** [comment]
This is a major:

1. For upgraded dependencies (which bring breaking changes)
2. Because you can no longer pass a custom store via `opts.db`. However, you can pass one as a function, which brings the signature closer to `levelup`:

```js
var jsondown = require('jsondown')
var level = require('level-test')(jsondown, { clean: true })
```

File: package.json [hunk 1]
```diff
@@ -8,36 +8,28 @@
...
     "rimraf": "~2.6.1",
     "xtend": "~4.0.0"
   },
   "optionalDependencies": {
-    "leveldown": "^1.2.2"
+    "leveldown": "~3.0.1"
   },
   "devDependencies": {
-    "tape": "~4.8.0"
+    "airtap": "0.0.4",
```
Comment (ralphtheninja) [inline package.json]: :100:

Comment (ralphtheninja) [reply package.json]: Testing the test framework in the browser :grinning:

**Comment by vweevers** [comment]
Opened a few issues for remaining tasks.

I will squash a few commits here, then merge.

**Pull request merged**